### ISSUE TITLE
[Bug] Listen.Moe in Fdroid Store restarts itself.

### ISSUE THREAD

**Issue by Ryu945** [LISTEN-moe/android-app#51]
**App constantly restarts itself.  I force quit it and I find it running again sometime later even though I have not used the app.  I also found that starting Work Profile will start up the app in the back ground even though the app has nothing to do with the Work Profile.**

**The play menus in notifications show up up whenever the app restarts itself**

- App version:  5.4.2  (From Fdroid Store)
- Android version: 11
- Device:  LG V20 
- Priority this should have – please be realistic and elaborate if possible:  Low priority since this is mostly a GUI nuisance and battery optimization problem.

**Comment by Ryu945** [comment]
This is still not fixed

**Comment by arkon** [comment]
Should be improved after #56
